sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -22,14 +22,24 @@
 """
 
 from collections import defaultdict
-from warnings import warn
+import numbers
 
+from apache_beam.metrics.cells import DistributionData
+from apache_beam.metrics.cells import DistributionResult
 from apache_beam.metrics.execution import MetricKey
 from apache_beam.metrics.execution import MetricResult
 from apache_beam.metrics.metric import MetricResults
 from apache_beam.metrics.metricbase import MetricName
 
 
+def _get_match(proto, filter_fn):
+  query = [elm for elm in proto if filter_fn(elm)]
+  if len(query) == 0:
+    raise ValueError('Could not find element')
+
+  return query[0]
```

Comment:
Is it an error to have more than 1 matching things? If the answer is yes, lets raise an error, otherwise documenting is fine.